sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -783,21 +783,22 @@ export class InstitutionService extends RecordDataModelService<Institution> {
   async updateInstitution(
     institutionId: number,
     auditUserId: number,
-    updateInstitution: Partial<UpdateInstitution>,
+    updateInstitution: UpdateInstitution,
```

Comment:
why `Partial<UpdateInstitution>1 was removed. is it for 
![image](https://user-images.githubusercontent.com/77353155/196527552-e8c5c6db-3f95-4797-a687-7ecf2472fc78.png)
?

Reply:
Yes. Partial was making all the properties in UpdateInstitution as optional.  
The mailing address and contact details are not optional and will always be present(It belongs to both institution and ministry update).